packages/web/src/components/createCell.tsx

```diff
@@ -111,103 +212,115 @@ const isDataEmptyArray = (data: DataObject) => {
-    // destructuring to not pass children to beforeQuery
-    const { children: _children, ...variables } = props
-
-    return (
-      <Query
-        query={
-          typeof QUERY === 'function' ? QUERY(beforeQuery(variables)) : QUERY
-        }
-        {...beforeQuery(variables)}
-      >
-        {({ error, loading, data, ...queryRest }) => {
-          if (error) {
-            if (Failure) {
-              return (
-                <Failure
-                  error={error}
-                  /**
-                   * error code
-                   * @optional
-                   * @type {string}
-                   * @see https://www.apollographql.com/docs/apollo-server/data/errors/#error-codes
-                   * The error code came from `error.graphQLErrors[0].extensions.code`
-                   */
-                  errorCode={
-                    error.graphQLErrors?.[0]?.extensions?.['code'] as string
-                  }
-                  {...{ updating: loading, ...queryRest, ...props }}
-                />
-              )
-            } else {
-              throw error
-            }
-          } else if (data) {
-            if (
-              typeof Empty !== 'undefined' &&
-              isEmpty(data, { isDataEmpty })
-            ) {
-              return (
-                <Empty
-                  {...{ ...data, updating: loading, ...queryRest, ...props }}
-                />
-              )
-            } else {
-              return (
-                <Success
-                  {...afterQuery(data)}
-                  {...{ updating: loading, ...queryRest, ...props }}
-                />
-              )
-            }
-          } else if (loading) {
-            return <Loading {...queryRest} {...props} />
-          } else {
-            console.warn(
-              `If you're using Apollo Client, check for its debug logs here in the console, which may help explain the error.`
-            )
-            throw new Error(
-              'Cannot render Cell: reached an unexpected state where the query succeeded but `data` is `null`. If this happened in Storybook, your query could be missing fields; otherwise this is most likely a GraphQL caching bug. Note that adding an `id` field to all the fields on your query may fix the issue.'
-            )
-          }
-        }}
-      </Query>
+  /**
+   * A Cell declaratively renders components based on the lifecycle of a GraphQL query.
+   *
+   * If the return's confusing, conider that a Cell's usually used in a Page like so:
+   *
+   * ```js
+   * import PostCell from 'src/components/PostCell'
+   *
+   * const PostsPage = ({ id }) => {
+   *   return <PostCell id={id} />
+   * }
+   * ```
+   *
+   * `PostCell` is actually this component.
+   */
+  function NamedCell(props: React.PropsWithChildren<CellProps>) {
+    /**
+     * Right now, Cells don't render `children`.
+     */
+    const { children: _, ...variables } = props
```

Comment:
May be a discussion for another time, but doing something like this seems conceivable to me:
```js
<PostsCell>
  <AddPost>
</PostsCell>
```
The difficult part about this is, where do we render it? My gut reaction is in both `<Empty />` and `<Success />`.

Reply:
Hmmm.... interesting.... @callingmedic911 and I had to do some workarounds for this (I think) when we did the filters on the tape dashboard.

Not needed for this PR, but we should bring up for discussion for sure.